sdk/core/core-rest-pipeline/test/throwRestError.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { it } from "vitest";
```

Comment:
We should describe the entire thing using `describe` and not just `it`

Reply:
This test is just for debugging so I can see the output and make sure it's sanitized. Will remove the file before merging

Reply:
Removed the test